.buildkite/pipelines/pull_request/base.yml

```diff
@@ -95,6 +95,19 @@ steps:
         - exit_status: '-1'
           limit: 3
 
+  - command: .buildkite/scripts/steps/functional/security_serverless.sh
```

Comment:
The other serverless suites are temporarily set to `soft_fail: true` on pull requests due to active development (they'll be switching to the serverless docker image in the near future).  They won't fail a PR build if they're flaky.

I don't have any concerns with merging this as is, but in the event this is flaky and builds are set to fail are we okay with operations skipping these tests, and if so can you share your preferences on how?

Reply:
One other note, if we do want these to fail in pull requests can we move the serverless.yml changes to `.buildkite/pipelines/on_merge.yml`?  That'll give us failed test reporting for tracking skips.

cc @oatkiller

Reply:
Thanks for the review @jbudz!! I completely missed the soft_fail. Can you please take a look again when you have the chance? 

About moving the changes to the `on_merge` I'm not quite sure, so let's see if @oatkiller gives feedback. If we have to do it, can be done in a follow-up PR?

Reply:
Follow-up PR works too.  LGTM

Reply:
Thanks all. on_merge makes sense to me. [I have a PR that should allow the `/skip` command to work for cypress tests going forward.](https://github.com/elastic/kibana/pull/162279) It may need some special configuration for this suite, I'll check